src/redux/slices/user/index.ts

```diff
@@ -101,7 +101,7 @@ export const registerUser = createAsyncThunk(
     }
 
     await setStorage(
-      null,
+      response.user,
```

Comment:
any idea why this was null before?

Reply:
At the time, the register endpoint did not return a user object.